cmd/kar-controllers/app/server.go

```diff
@@ -86,3 +91,15 @@ func listenHealthProbe(opt *options.ServerOption) error {
 
 	return nil
 }
+
+// Starts the readiness probe listener
+func listenReadinessProbe(opt *options.ServerOption) error {
```

Comment:
I don't think we want to start a second HTTP server. The `readyz` handler should be added to the existing probe server.